test/test_holidays.rb

```diff
@@ -118,6 +121,10 @@ def test_any_region_holiday_next
     holidays = Holidays.next_holidays(2, [:any], Date.civil(2008,5,1))
     assert_equal 2, holidays.length
 
+    # Should return 1 holiday in July
+    holidays = Holidays.next_holidays(1, [:jp], Date.civil(2016, 5, 22))
+    assert_equal ['2016-07-18','海の日'] , [holidays.first[:date].to_s, holidays.first[:name].to_s]
+
```

Comment:
Thank you for adding jp region test!! The probrem will not never appear in the future. 😄